v2/bundle/jwtbundle/bundle_test.go

```diff
@@ -220,38 +221,30 @@ func TestJWTKeys_crud(t *testing.T) {
 
 func TestMarshal(t *testing.T) {
 	// Load a bundle to marshal
-	bundle, err := Load(td, "testdata/jwks_valid_1.json")
+	bundle, err := jwtbundle.Load(td, "testdata/jwks_valid_2.json")
 	require.NoError(t, err)
 
 	// Marshal the bundle
 	bundleBytesMarshal, err := bundle.Marshal()
 	require.NoError(t, err)
 	require.NotNil(t, bundleBytesMarshal)
 
-	// Load original bytes for comparison
-	bundleBytesFile, err := ioutil.ReadFile("testdata/jwks_valid_1.json")
+	// Prase the marshalled bundle
```

Comment:
We should probably stick with `marshaled` instead of `marshalled` (to match what is used by the Go stdlib)
```suggestion
	// Parse the marshaled bundle
```